src/pages/cart/CartItem.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './CartItem.scss';
+
+const Cartitem = ({ idx, item, total }) => {
+  return (
+    <tr className="cartItem">
+      <td className="checkBox">
+        <input type="checkbox" />
+      </td>
+      <td>
+        <img src={item.product} alt={`${item.info}`} />
```

Comment:
item.info가 string이라면 템플릿 리터럴 안쓰고 그대로 넣어줘도 될 것 같네요!
```suggestion
        <img src={item.product} alt={item.info} />
```